src/components/NavigationBarBrand/NavigationBarBrand.component.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Styled from './NavigationBarBrand.styled';
+
+function NavigationBarBrand() {
+  return (
+    <Styled.Container className="navbar-brand" href="/" data-testid="NavigationBarBrand">
+      <Styled.Image
+        src="./logo120.png"
+        alt=""
+        width="30"
```

Comment:
Might be better to add width and height to the styles instead of attributes. Just a general best practice.